draft-sheffer-wimse-s2s-protocol.md

```diff
@@ -52,34 +52,83 @@ TODO Abstract
 
 # Introduction
 
-TODO Introduction
+This document defines authentication and authorization in the context of interaction between two workloads.
+This is the core component of the WIMSE architecture {{?I-D.ietf-wimse-arch}}.
+Assume that Service A needs to call Service B. For simplicity, this document focuses on REST services,
```

Comment:
I think we should stick to one term - either `workload` or `service`. Using these two interchangeably may confuse readers.

Reply:
Agreed.

Reply:
On second thoughts, even in the group charter we called it "service to service". I will let others chime in.

Reply:
I'm in favor of switching to workload to workload vs service.  I feel that the workload term is more generic and its what we are using elsewhere.  I believe the way we are using the they are synonymous.